lua/starfall/libs_sv/entities.lua

```diff
@@ -731,24 +729,24 @@ function ents_methods:getAllConstrained(filter)
 			if not filter or filter.Parent then
 				local parent = ent:GetParent()
 				if parent then recursive_find(parent) end
-				for k, child in pairs(ent:GetChildren()) do
+				for _, child in pairs(ent:GetChildren()) do
 					recursive_find(child)
 				end
 			end
 			if not filter or filter.Wire then
 				if istable(ent.Inputs) then
-					for k, v in pairs(ent.Inputs) do
+					for _, v in pairs(ent.Inputs) do
 						if isentity(v.Src) and v.Src:IsValid() then
 							recursive_find(v.Src)
 						end
 					end
 				end
 				if istable(ent.Outputs) then
-					for k, v in pairs(ent.Outputs) do
+					for _, v in pairs(ent.Outputs) do
 						if istable(v.Connected) then
-							for k, v in pairs(v.Connected) do
-								if isentity(v.Entity) and v.Entity:IsValid() then
-									recursive_find(v.Entity)
+							for _, v2 in pairs(v.Connected) do
+								if isentity(v2.Entity) and v2.Entity:IsValid() then
+									recursive_find(v2.Entity)
```

Comment:
I think for these I prefer the local shadowing

Reply:
To each their own, reverted those ones.